### DISCUSSION TITLE
Connection terminated unexpectedly

### DISCUSSION THREAD

**Discussion by KingDracula** [sequelize/sequelize#15371] [Help & Questions]
i was used with postgres.
some times occasionally this error 
 Connection terminated unexpectedly  

but  retry the api it is work .
this is the config 
   "dialectOptions": {
            "useUTC": true,
            "timezone": "UTC",
            "connectionTimeout": 60000,
            "requestTimeout": 60000
        },
        "pool": {
            "max": 100,
            "min": 5,
            "idle": 300000,
            "acquire": 300000
        },
        "idleTimeoutMillis": 0,
        "connectionTimeoutMillis": 0,

**Comment by tejasautkar** [comment]
After facing similar issue, we added this config
`retry: {
match: [Sequelize.ConnectionError],
max: 2
}`

This will retry the connection twice if the error matches ConnectionError.
